secure-api-gateway-ob-uk-common-obie-datamodel/src/test/java/uk/org/openbanking/testsupport/v4/vrp/OBDomesticVrpConsentRequestTestDataFactory.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright © 2020-2024 ForgeRock AS (obst@forgerock.com)
```

Comment:
I asked Alin too, but why are we using 2020-2024 for new files (not just 2024)?

Reply:
I am really not sure about this but this is how the copyright template has been implemented initially and it's enforced by the pipelines to be exactly under this specific format for all the files, new or old. We can discuss this further with @BohoCode

Reply:
Sure, ok... thanks for that info.

Just asking as with IG core, new files have just the current year (extended whenever the file is touched thereafter).